person.rb

```diff
@@ -1,9 +1,9 @@
-require_relative 'nameable'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative './nameable'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
 
 class Person < Nameable
```

Comment:
- [ ] It seems you forgot to implement the "A person has many rentals" relationship. Kindly include it.

![image](https://user-images.githubusercontent.com/86318284/195913607-b36c703b-10a9-466a-8c58-3fb88a56c19d.png)

Reply:
Hi @Nemwel-Boniface 
Please refer to my comment on the first change requested in this PR